server/src/controller/users.ts

```diff
@@ -0,0 +1,18 @@
+import express from "express";
+
+import { getUserById } from "../service/user";
+import { get } from "lodash";
+
+export async function getUser(req: express.Request, res: express.Response) {
+  try {
+    const id = get(req, "identity._id");
+
+    const user = await getUserById(id);
+
+    if (id) return res.status(200).json(user);
+
+    return res.status(404).json({ message: "User not found" });
```

Comment:
please don't use reverse logic :D if the user doesn't exist return error message, otherwise return the user :D